src/Controller/ConfigController.php

```diff
@@ -341,14 +345,79 @@ public function getAction(Request $request, Service $graphQlService): JsonRespon
-        foreach ($specialSettings as $key) {
-            if (!isset($config['schema']['specialEntities'][$key])) {
-                $config['schema']['specialEntities'][$key] = ["id" => $key];
-            }
-        }
+        $coreSettings = [
+            new SpecialEntitySetting(
+                'document',
+                true,
+                false,
+                false,
+                false,
+                $config['schema']['specialEntities']['document']['read'] ?? false,
+                $config['schema']['specialEntities']['document']['create'] ?? false,
+                $config['schema']['specialEntities']['document']['update'] ?? false,
+                $config['schema']['specialEntities']['document']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'document_folder',
+                true,
+                false,
+                false,
+                false,
+                $config['schema']['specialEntities']['document_folder']['read'] ?? false,
+                $config['schema']['specialEntities']['document_folder']['create'] ?? false,
+                $config['schema']['specialEntities']['document_folder']['update'] ?? false,
+                $config['schema']['specialEntities']['document_folder']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'asset',
+                true,
+                true,
+                true,
+                true,
+                $config['schema']['specialEntities']['asset']['read'] ?? false,
+                $config['schema']['specialEntities']['asset']['create'] ?? false,
+                $config['schema']['specialEntities']['asset']['update'] ?? false,
+                $config['schema']['specialEntities']['asset']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'asset_folder',
+                true,
+                true,
+                true,
+                true,
+                $config['schema']['specialEntities']['asset_folder']['read'] ?? false,
+                $config['schema']['specialEntities']['asset_folder']['create'] ?? false,
+                $config['schema']['specialEntities']['asset_folder']['update'] ?? false,
+                $config['schema']['specialEntities']['asset_folder']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'asset_listing',
+                true,
+                true,
+                true,
+                true,
+                $config['schema']['specialEntities']['asset_listing']['read'] ?? false,
+                $config['schema']['specialEntities']['asset_listing']['create'] ?? false,
+                $config['schema']['specialEntities']['asset_listing']['update'] ?? false,
+                $config['schema']['specialEntities']['asset_listing']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'object_folder',
+                true,
```

Comment:
Could you please recheck the matrix? I don't see why CUD should not be possible for object folders.

Reply:
done